spec/spec_helper.rb

```diff
@@ -46,6 +46,10 @@ class CashRegister < ActiveRecord::Base
     honeywell "HON"
     sharp "SHCAY"
   end
+
+  attr_accessor :include_validations, :required_attribute
+
+  validates_presence_of :required_attribute, if: -> { include_validations }
```

Comment:
Not sure if this is actually more readable or not, but, an alternative here is to subclass `CashRegister` as `ValidatedCashRegister` or something in order to avoid carrying around the `include_validations` value. I think this might make the class more exemplary of what a real class using flexible_enum would look like.